README.md

```diff
@@ -123,13 +123,11 @@ $ [npm run / yarn] preact build
     --cwd              A directory to use instead of $PWD  (default .)
     --sw               Generate and attach a Service Worker  (default true)
     --babelConfig      Path to custom Babel config (default .babelrc)
-    --json             Generate build stats for bundle analysis
-    --template         Path to custom HTML template (default 'src/template.html')
-    --analyze          Launch interactive Analyzer to inspect production bundle(s)
     --prerender        Renders route(s) into generated static HTML  (default true)
     --prerenderUrls    Path to pre-rendered routes config  (default prerender-urls.json)
-    --brotli           Adds brotli redirects to the service worker  (default false)
-    --inline-css       Adds critical css to the prerendered markup  (default true)
+    --template         Path to custom HTML template (default 'src/template.html')
+    --inlineCss        Adds critical css to the prerendered markup  (default true)
+    --analyze          Launch interactive Analyzer to inspect production bundle(s) (default false)
```

Comment:
`--template` and `--analyze` were just moved to match the order for `preact watch`'s flags -- ordering consistency is just a nice-to-have and I felt dev's order was better.